test/utils/expose_keptn_bridge.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
 
 echo "Exposing Keptn Bridge..."
+kubectl -n keptn delete secret bridge-credentials
 # expose Keptn's Bridge (for easier troubleshooting/debugging afterwards)
 kubectl -n keptn create secret generic bridge-credentials --from-literal="BASIC_AUTH_USERNAME=${NIGHTLY_BRIDGE_USERNAME}" --from-literal="BASIC_AUTH_PASSWORD=${NIGHTLY_BRIDGE_PASSWORD}"
-keptn configure bridge --action=expose
 # restart bridge pod to make use of the secret
 kubectl -n keptn delete pods --selector=run=bridge
 sleep 5
+kubectl patch svc bridge -n keptn -p '{"spec": {"type": "LoadBalancer"}}'
+sleep 10
```

Comment:
In this test, we could re-use the already existing generated `bridge-credentials` secret. Furthermore, the service of the bridge does not require a LoadBalancer.